sdfx_st/hal/inc/device.h

```diff
@@ -33,8 +33,15 @@
 #define MBED_DEVICE_H
 
 //=======================================
-#define DEVICE_ID_LENGTH       24
+ #define DEVICE_ID_LENGTH       24
```

Comment:
```suggestion
#define DEVICE_ID_LENGTH       24
```

This applies to every new line in this file.

Reply:
[Done](https://github.com/MCU-Driver-HAL/MCU-Driver-ST/compare/5ae9d7b92278fb06f5f22dfb7733b3e2e030c95a..05ba2dd1d32f7ba9dc49a2d03068861f30908cea).